fieldsolver/gridGlue.cpp

```diff
@@ -179,6 +189,244 @@ void feedMomentsIntoFsGrid(dccrg::Dccrg<SpatialCell,dccrg::Cartesian_Geometry>&
 
   MPI_Waitall(sendRequests.size(), sendRequests.data(), MPI_STATUSES_IGNORE);
 
+  // ---------------------------------------------------------------------------------
+  // Boxcar Smoothening
+  const int *mntDims= &momentsGrid.getLocalSize()[0];
+  std::array<Real,fsgrids::moments::N_MOMENTS> *Grid;
+  std::array<Real,fsgrids::moments::N_MOMENTS> *GridL1;
+  std::array<Real,fsgrids::moments::N_MOMENTS> *GridL2;
+  std::array<Real,fsgrids::moments::N_MOMENTS> *GridR1;
+  std::array<Real,fsgrids::moments::N_MOMENTS> *GridR2;
+  std::array<Real,fsgrids::moments::N_MOMENTS> *swap;
+  auto swapGrid= momentsGrid;
```

Comment:
@ursg would `FsGrid< std::array<Real, fsgrids::moments::N_MOMENTS>, 2> swapGrid = momentsGrid;` here work? Or with `= momentsGrid` in the next line? Is there any difference or prefernece vs. `auto`?

Reply:
In principle, both should work just fine.
But for readability, I guess it's better to actually write the whole type (```FsGrid< std::array<Real, fsgrids::moments::N_MOMENTS>, 2>```) there, since otherwise people reading the code might have no idea what's going on. :)

Reply:
Ok thanks!